README.md

```diff
@@ -22,9 +22,15 @@ directly without another `pip install`.
 
 ### CommCare HQ OAuth Integration
 
-- Create an OAuth application on CommCare HQ using Django Admin
-  `<hq_host>/admin/oauth2_provider/application/`. Use
-  `<superset_host>/oauth-authorized/commcare` as the redirect URL.
+- Create an OAuth application on CommCare HQ using Django Admin at the URL
+  `<hq_host>/admin/oauth2_provider/application/` with the following settings:
+  - Redirect URIs: `<superset_host>/oauth-authorized/commcare`
+  - Leave "Post logout redirect URIs" empty.
+  - Client type: Confidential
+  - Authorization grant type: Client credentials
+  - Give your OAuth application a name, like "CommCare Analytics" or "HQ Superset"
```

Comment:
nit: the default we have in the [config example file](https://github.com/dimagi/hq_superset/blob/61c78dd28ae3a99ede53bf3157c6c17bfa815e6a/superset_config.example.py#L23) is "commcare".
Good to mention that as well or as the recommendation to avoid unexpected issues.

Reply:
The two names refer to different things. This name is used by HQ to tell the user what application is requesting authorization (i.e. CommCare Analytics). 3075096ca51ed7efc5394d46f74fd6b79503d287 clarifies this. The name in hq_superset's `superset_config.py` identifies the OAuth provider (i.e. the CommCare HQ instance).